### ISSUE TITLE
(Question NOT Issue) How can I envoke scroll to top

### ISSUE THREAD

**Issue by willemevenwel** [HermanNygaard/react-scroll-to-top#4]
Say for instance a user has scrolled to the bottom, clicks 'Save', and after save, one would like to scroll the person to the top automatically (without click). How can I do this?

**Comment by HermanNygaard** [comment]
I don't think this package should provide such a function, at least not for now.
